packages/admin-ui-core/src/values/styleKit.ts

```diff
@@ -34,15 +44,21 @@ export function listBoxItem(tone: 'main' | 'critical', selected = false) {
   }
 }
 
-export function focusVisible(tone: Tone): StyleProp {
+export function focusVisible(
+  toneOrColorToken: Tone | ColorTokens,
```

Comment:
Color tokens are only meant to be used by a design token and not directly. You can think that `$bg.feedback` can be any color in different themes, but green will always be green, so using the colors this way is not scalable

Reply:
```suggestion
  tone: Tone,
```

Reply:
Roger that! I'm working on creating a design token instead.

Reply:
Done!